clusterdock/models.py

```diff
@@ -117,8 +117,13 @@ def start(self, network, pull_images=False, update_etc_hosts=True):
                 raise DuplicateHostnamesError(duplicates=duplicate_hostnames,
                                               network=self.network)
 
+        if clusterdock_args and clusterdock_args.clusterdock_config_directory:
+            self.clusterdock_config_host_dir = os.path.expanduser(clusterdock_args.clusterdock_config_directory)
```

Comment:
Use `os.path.realpath` also - here and below.

Reply:
Done.